packages/sdk/src/global-account/react/utils/notificationsAPI.ts

```diff
@@ -0,0 +1,305 @@
+import { debugB3React } from "@b3dotfun/sdk/shared/utils/debug";
+
+const debug = debugB3React("notificationsAPI");
+
+const API_URL = "https://notifications.b3.fun";
+
+export interface NotificationChannel {
+  id: number;
+  channel_type: "email" | "telegram" | "discord" | "sms" | "whatsapp" | "in_app";
+  enabled: number;
+  channel_identifier: string;
+}
+
+export interface UserData {
+  user: {
+    id: number;
+    user_id: string;
+  };
+  channels: NotificationChannel[];
+  appSettings: Array<{
+    app_id: string;
+    notification_type: string;
+    enabled: number;
+    channels: string;
+  }>;
+}
+
+export interface NotificationSettings {
+  notificationType: string;
+  channels: string[];
+}
+
+export const notificationsAPI = {
```

Comment:
Maybe I just haven't gotten there yet, but is there a listener for notifications where we can trigger a notification in B3 API and the user sees it in B3 SDK? Or is it all just device notifications? 

Also, let's say that a new partner, AGS - has users subscribing to notifications. Where are the docs to show the how to send one for their partner ID?